applications/metaquity_nft_provenance_tracker.md

```diff
@@ -0,0 +1,148 @@
+| -----: | ----------- | ------------- |
+| **0a.** | License |   Apache 2.0 |
+| **0b.** | Documentation | Architectural documentation of NFT-scanner |
+| 1. | Data model definition | Designing and defining the architectural overview of the data model for this application  |
+| 2. | NFT scanner module | This module will scan the blockchain for any NFT-related data from the genesis block and invoke the indexer module |
+
+### Milestone 2 
+
+- **Estimated Duration:** 1,5 month
+- **FTE:**  1,5
+- **Costs:** 10,000 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0  |
+| **0b.** | Documentation | A detailed developer documentation of the SDK (will be in gitbook format) and integration guide for other projects. |
+| 1. | NFT indexer module | Index all the data scanned by the scanner module and pass it to the database layer. |
+| 2. | Design and development of the NFT provenance SDK | A typescript-based SDK which consists of API calls to the database to fetch the NFT lifecycle. |
+| 3. | Technical Documentation for the SDK | We will provide a technical doc outlining integrating the SDK into other projects. |
+
+## Future Plans
+
+- Support the NFTs of Statemint
+- Support wasm based NFTs
+- Adding capabilities for tracking and tracing fractional ownership of NFTs
+- Adding capabilities for tracking locking/unlocking of NFTs
+- 
+
+## Additional information 
+
+ > When you talk about NFTs, do you mean the NFTs pallet, PSP34s, uniques pallet, or EVM-compatible ones?
+
+We will not be focusing currently on the EVM-compatible ones but stick to more Polkadot ecosystem NFT's which are powered mostly by the Uniques Pallet and going forward the NFT's pallet will be used. So this will be a tool for aggregation for NFTs created through both pallets.
+
+> Your tech stack mentions Elasticsearch, AWS, and nginx, but none of your deliverables touches those. How will I be able to replicate your results?
+
+I think there's some confusion related to this point. The deliverables are geared towards building and delivering the individual modules which will eventually be released as an SDK which other projects can integrate. This project will be a proof of concept (PoC) to demonstrate this particular use case and how this can be leveraged by other projects and parachains in the Polkadot ecosystem. Since this will be an SDK, anyone will be able to integrate this within their application. This acts as an enabler for this particular use case for provenance tracking of NFTs and fractional fungible tokens associated with those NFTs. This can extend to not only digital assets but also real-world assets (which is what is the primary focus of Metaquity Network). This grant application is in relation to taking this project to the proof of concept stage. The ancillary tech stack mentioned such as Elasticsearch, AWS, and others shall be implemented when we take this project to the production stage.
```

Comment:
Thanks for copying your answers to the proposal document. However, I think it'd be more intuitive if you directly integrated these answers into the proposal. For example, for the tech you could directly integrate them into the deliverables.